bin/get-google-analytics-pageviews

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+require "csv"
+require "google/apis/analyticsreporting_v4"
+require "googleauth"
+require "yaml"
+
+GA_REPORTING = Google::Apis::AnalyticsreportingV4
+PAGEVIEWS = "ga:dimension2"
+RENDERING_APP = "ga:dimension20"
+
+def load_config_data
+  YAML.load(File.open("config/ga_env.yml")).each do |key, value|
+    ENV[key.to_s] = value
+  end
+end
+
+def matrix_file
+  Dir["**"].select { |f| f =~ /\Amatrix_\d{4}-\d{2}-\d{2}_\d{2}:\d{2}:\d{2}.csv/ }.sort.last
+end
+
+def get_google_analytics_views(document_type, rendering_app)
+  return 0 unless document_type && rendering_app
+  service = ga_authorization
+
+  report_request = create_ga_report_request(document_type, rendering_app)
+  get_reports_request = get_reports_request(report_request)
+
+  response = service.batch_get_reports(get_reports_request)
+
+  views = response.reports[0].data.totals[0].values[0]
+  puts "Document type: #{document_type}, Rendering app: #{rendering_app}, Views:#{views}"
+  views
```

Comment:
Similar to the comment above, if an error occurs here do we get a 0 or an empty string?